core/src/main/java/com/graphhopper/reader/dem/CGIARProvider.java

```diff
@@ -80,84 +66,31 @@ public static void main(String[] args) {
 
         // 337.0
         System.out.println(provider.getEle(49.949784, 11.57517));
-        // 453.0
+        // 466.0
```

Comment:
These values seem to have changed, maybe the CGIAR data changed over time, maybe there was a CGIAR update? This happens for me with the current master as well, so it wasn't introduced by a change in this PR.